### DISCUSSION TITLE
Where is the posts / comments database?

### DISCUSSION THREAD

**Discussion by chicchera** [MalloyDelacroix/DownloaderForReddit#287] [Q&A] [answered]
I need to work with the extracted data but I cannot find the databases, and the download directories are empty.
Yet, when I select Database from the menu I can see the data previously downloaded.

**Comment by MalloyDelacroix** [comment, marked as answer]
On windows the database file is at "C:/Users/[username]/AppData/Roaming/SomeGuySoftware/DownloaderForReddit".  On Linux its at "~/.SomeGuySoftware/DownloaderForReddit".

**Reply by chicchera** [reply]
Thanks for the prompt replay. I just didn't recognize it because I cannot find the structure of the tables. I found a better interface, without going into the command line, and now I see everything.
